ambari-server/src/test/java/org/apache/ambari/server/controller/internal/ExportBlueprintRequestTest.java

```diff
@@ -49,69 +47,113 @@
 import org.apache.ambari.server.topology.Blueprint;
 import org.apache.ambari.server.topology.HostGroup;
 import org.apache.ambari.server.topology.HostGroupInfo;
-import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.powermock.api.easymock.PowerMock;
 import org.powermock.core.classloader.annotations.PrepareForTest;
 import org.powermock.modules.junit4.PowerMockRunner;
 
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+
 /**
  * ExportBlueprintRequest unit tests.
  */
-@SuppressWarnings("unchecked")
 @RunWith(PowerMockRunner.class)
-@PrepareForTest({ExportBlueprintRequest.ExportedHostGroup.class})
+@PrepareForTest({ InetAddress.class })
 public class ExportBlueprintRequestTest {
```

Comment:
According to powermock docs, when mocking system classes, the class that calls the system class should be prepared.

https://github.com/powermock/powermock/wiki/Mock-System

Reply:
Good point.